### PR TITLE
Integration of the Animal classification model

### PR DESCRIPTION
**Pull request by NikitaVishnyak**

### CODE REVIEW

File: web/models.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 db_name = config('SQL_DB')
 
 app = Flask(__name__)
-app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://{}:{}@db/{}'.format(db_user, db_password, db_name)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://{}:{}@localhost/{}'.format(db_user, db_password, db_name)
```
Comment (cartiss) [inline web/models.py:10]: Return to db for docker

```suggestion
app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://{}:{}@db/{}'.format(db_user, db_password, db_name)
```

File: web/web.py [hunk 2]
```diff
@@ -2,22 +2,27 @@
 import pathlib
 import sys
 from decouple import config
+import numpy as np
 
 from flask import render_template, request
 
+
+
```
Comment (cartiss) [inline web/web.py:10]: Too much spaces here, delete please

File: web/web.py [hunk 3]
```diff
@@ -2,22 +2,27 @@
 import pathlib
 import sys
 from decouple import config
+import numpy as np
```
Comment (cartiss) [inline web/web.py:5]: Make correct import order

https://peps.python.org/pep-0008/#imports

File: web/web.py [hunk 4]
```diff
@@ -2,22 +2,27 @@
...
+
+
 PARENT_FOLDER = str(pathlib.Path(__file__).parent.parent.absolute())
 sys.path.insert(0, PARENT_FOLDER)
+from models.animals_classification.model import Predictor
 
 # flake8: noqa
 from logic import make_sentiment_prediction, make_disaster_prediction, compile_results, make_homepage_queryset, \
     make_category_projects_queryset
 # flake8: noqa
```
Comment (cartiss) [inline web/web.py:18]: For what do you need here noqa

File: web/web.py [hunk 5]
```diff
@@ -2,22 +2,27 @@
...
 
 from flask import render_template, request
 
+
+
 PARENT_FOLDER = str(pathlib.Path(__file__).parent.parent.absolute())
 sys.path.insert(0, PARENT_FOLDER)
+from models.animals_classification.model import Predictor
 
 # flake8: noqa
```
Comment (cartiss) [inline web/web.py:15]: For what do you need here noqa

**Review by cartiss — CHANGES_REQUESTED** [review]

File: web/web.py (hunk 4 above)
Comment (NikitaVishnyak) [reply web/web.py:18]: We need it because of flake 8. It gives error " module level import not at top of file".

Comment (cartiss) [reply web/web.py:18]: Turn off exact flake8 parameter at the top of the file.
With flake8: noqa you disable full flake8 for the whole file

File: web/web.py [hunk 6]
```diff
@@ -1,23 +1,24 @@
 """Run Flask application."""
-import pathlib
 import sys
+import pathlib
 from decouple import config
```
Comment (cartiss) [inline web/web.py:4]: from .... must be after import ....

**Review by cartiss — CHANGES_REQUESTED** [review]

**Review by cartiss — APPROVED** [review]

**Pull request merged**